mcd.graphql

```diff
@@ -157,12 +157,12 @@ type FlipEvent {
   tx:      Tx        #                desc: transaction meta
 }
 
-# Pit.file(ilk _).call   -> create FileEvent id: e.ilk
-# Pit.file(what _).call  -> create FileEvent id: e.address
-# Cat.file(ilk _).call   -> create FileEvent id: e.ilk
-# Cat.file(what _).call  -> create FileEvent id: e.address
-# Drip.file(ilk _).call  -> create FileEvent id: e.ilk
-# Drip.file(what _).call -> create FileEvent id: e.address
+# Vat.file(ilk _).call  -> create FileEvent id: e.ilk
+# Vat.file(what _).call -> create FileEvent id: e.address
+# Cat.file(ilk _).call  -> create FileEvent id: e.ilk
+# Cat.file(what _).call -> create FileEvent id: e.address
+# Jug.file(ilk _).call  -> create FileEvent id: e.ilk
+# Jug.file(what _).call -> create FileEvent id: e.address
```

Comment:
It looks like this might be referring to two different `file` functions on the `Jug` - one that sets the `base`, and another that sets the `vow`. However, perhaps it only refers to the one that sets the `vow` (since that's the only one that takes an `address`)? Either way, perhaps worth clarifying exactly which `file` events we're referring to (both vs just `Jug.file(bytes32,address`)

Reply:
Agreed that we could use some clarification here, what would be the best way of doing that? Is that something that would be specified in the comments?